magellan-library/src/main/java/com/wealthfront/magellan/navigation/NavigableListener.kt

```diff
@@ -5,12 +5,13 @@ import com.wealthfront.magellan.lifecycle.LifecycleAware
 
 interface NavigableListener : LifecycleAware {
 
-  fun onNavigate() {}
-
   fun onNavigableShown(navigable: NavigableCompat) {}
 
   fun onNavigableHidden(navigable: NavigableCompat) {}
 
+  @JvmDefault
```

Comment:
Why is this the only one marked with `@JvmDefault`?

Reply:
I supposed we would always want to implement `onNavigableShown ` and `onNavigableHidden `. But i suppose not.